carbon-design-system/carbon-addons-iot-react
# Dynamic aggregation part2

Closes #3533

**Summary**

Updated logic to a call back for showing the DataSeriesFormItemModal 
Added a add Aggregations button to the modal

**Change List (commits, features, bugs, etc)**

CardEditForm.jsx
CardEditFormContent.jsx
DataSeriesFormItemModal.jsx
DataSeriesFormItemModal.test
DataSeriesFormItemContent.jsx
DataSeriesFormItemContent.test
ImageCardFormContent.jsx
TableCardFormContent.jsx
TableCardFormSettings.jsx
TableCardFormSettings.test
CardEditor.jsx
CardEditor.Story
Dashboard.Story
DashboardEditor.jsx
DashboardEditor.story
DashboardEditor.test
EditorUtils.jsx
EditorUtils.test
HotspotEditorDataSourceTab.jsx
HotspotEditorDataSourceTab.test
HotspotEditorDataSourceTab.story
HotspotEditorModal.jsx
HotspotEditorModal.test
HotspotEditorModal.story
HotspotEditorTooltipTab.story.jsx
HotspotTextStyleTab.story.jsx


**Test**
- Check all Dashboard and CardEditor stories 
- Check V2 Pressure story Edit a TimeSeries card and select V2 pressure you should see the add aggregation method button.

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Check all Dashboard and CardEditor stories 



**Things to look for during review**
